sdk/python/kfp/_client.py

```diff
@@ -74,6 +74,8 @@ def camel_case_to_snake_case(name):
 KF_PIPELINES_DEFAULT_EXPERIMENT_NAME = 'KF_PIPELINES_DEFAULT_EXPERIMENT_NAME'
 KF_PIPELINES_OVERRIDE_EXPERIMENT_NAME = 'KF_PIPELINES_OVERRIDE_EXPERIMENT_NAME'
 
+LOCAL_KFP_CONTEXT = os.path.expanduser('~/.config/kfp/context.json')
```

Comment:
This is evaluated at the time of the kfp package import.
I think it's better to execute any code (e.g. expanduser) closer to the usage site.

Reply:
moved to inside the Client class.